src/models/git-hub-server-app.ts

```diff
@@ -0,0 +1,38 @@
+import { Model, DataTypes } from "sequelize";
+import { encrypted, sequelize } from "models/sequelize";
+
+class GitHubServerApp extends Model {}
+
+GitHubServerApp.init({
+	id: {
+		type: DataTypes.INTEGER,
+		primaryKey: true,
+		allowNull: false,
+		autoIncrement: true
+	},
+	uuid: {
+		type: DataTypes.STRING,
+		unique: true
+	},
```

Comment:
I believe that sequelize has a UUID datatype and can create a default value on model creation.  Also, you need to set the `allowNull: false` here.

```
uuid: {
    type: DataTypes.UUID,
    defaultValue: Sequelize.UUIDV4,
   unique: true,
   allowNull: false
}
```